operator/src/main/java/oracle/kubernetes/operator/DomainStatusUpdater.java

```diff
@@ -570,6 +572,22 @@ DomainStatusUpdaterContext createContext(Packet packet) {
     void modifyStatus(DomainStatus domainStatus) { // no-op; modification happens in the context itself.
     }
 
+    @Override
+    public NextAction apply(Packet packet) {
+      return shouldSkipDomainStatusUpdate(packet)
+              ? doNext(getNext().getNext(), packet) : super.apply(packet);
+    }
+
+    private boolean shouldSkipDomainStatusUpdate(Packet packet) {
+      boolean isDomainResourcesValidation =
```

Comment:
This might be clearer if you used getOrDefault()... Something like:
Boolean idDomainResourcesValidation = (Boolean) packet.getOrDefault(DOMAIN_RESOURCES_VALIDATION, Boolean.FALSE);

Reply:
Thanks.  Great suggestion. Done.